### ISSUE TITLE
can not use 'ref' on styled-components

### ISSUE THREAD

**Issue by ppierre** [michaelauderer/stencil-styled-components#4]
When there is a 'ref' attribute on a 'styled-component', it isn't passed down to the underlying 'h' call.

    stencil-styled-components/lib/create-element.js:17
    -      return h(Tag, { class: componentId, }, children)
    +      return h(Tag, { class: componentId, ref:props.ref }, children)

**Comment by michaelauderer** [comment]
Thank you, @ppierre!

**Comment by dangodev** [comment]
👋 Hey just got this error from `1.3.4`: 

```
TypeError: Cannot read property 'ref' of null
```

From this line:

```
      return h(Tag, { class: componentId, ref: props.ref }, children)
```

I guess `props` is undefined in this instance?

**Comment by ppierre** [comment]
`return h(Tag, { class: componentId, ref: (props && props.ref) }, children)`

Sorry I run into that 'ref' wasn't working. I have just searched a kick fix and suggested it. Same for this proposition.

**Comment by michaelauderer** [comment]
@dangodev @ppierre 

Fixed and published in 1.3.5!

I'm too busy at the moment but hoping someone could write some tests soon :-)
